WalletWasabi.Fluent/CrashReport/CrashReporter.cs

```diff
@@ -0,0 +1,73 @@
+using System;
+using System.Diagnostics;
+using WalletWasabi.Logging;
+using WalletWasabi.Microservices;
+using WalletWasabi.Models;
+
+namespace WalletWasabi.Fluent.CrashReport
+{
+	public static class CrashReporter
+	{
+		public static bool TryInvokeIfRequired(Exception exceptionToReport)
+		{
+			if (exceptionToReport is null)
```

Comment:
Either this is extra, or you should use: `Exception? exceptionToReport` for the parameter.

Reply:
Removed the check. You should not call this function with null